File: src/main/kotlin/io/github/dqualizer/dqlang/types/dam/architecture/apischema/field/Output.java
package io.github.dqualizer.dqlang.types.dam.architecture.apischema.field;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.*;

@Getter
@ToString
@AllArgsConstructor
@NoArgsConstructor
@Data
public class Output {

    private String type;
    @JsonProperty("expected_code")
    private String expectedCode;
}

File: src/main/kotlin/io/github/dqualizer/dqlang/archive/openapiadapter/dqlang/field/Output.kt
package io.github.dqualizer.dqlang.archive.openapiadapter.dqlang.field

import com.fasterxml.jackson.annotation.JsonProperty
import lombok.AllArgsConstructor
import lombok.Getter
import lombok.NoArgsConstructor
import lombok.ToString

@Getter
@ToString
@AllArgsConstructor
@NoArgsConstructor
class Output {
    val type: String? = null

    @JsonProperty("expected_code")
    val expectedCode: String? = null
}
